lib/utils/exec.go

```
@@ -128,6 +131,27 @@ func RunStream(ctx context.Context, w io.Writer, args ...string) error {
 	return trace.Wrap(cmd.Wait())
 }
 
+// ExecUnprivileged executes the specified command as unprivileged user
+func ExecUnprivileged(ctx context.Context, command string, args []string, opts ...CommandOptionSetter) error {
+	nobody, err := user.Lookup("nobody")
+	if err != nil {
+		return trace.ConvertSystemError(err)
+	}
+	cmd := exec.CommandContext(ctx, command, args...)
+	uid, _ := strconv.Atoi(nobody.Uid)
```

Do we have to ignore the error here and below?